biogrid/libs/grid-simulator/src/lib/validate-inputs/validate-inputs.ts

```diff
@@ -0,0 +1,26 @@
+// Validation
```

Comment:
this is awesome. We should all implement this in

Reply:
nit: @aosmann7 let's us all implement the validation for the inputs we are getting

Reply:
+1 this class is awesome